### ISSUE TITLE
null blueprint in WorkController causes exception in __repr__

### ISSUE THREAD

**Issue by benkehoe** [celery/celery#2514]
Adding the following signal callback:
```
@worker_init.connect
def init(*args, **kwargs):
	print args, kwargs
```

causes the following exception:

```
File "/usr/local/bin/celery", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python2.7/dist-packages/celery/__main__.py", line 30, in main
    main()
  File "/usr/local/lib/python2.7/dist-packages/celery/bin/celery.py", line 81, in main
    cmd.execute_from_commandline(argv)
  File "/usr/local/lib/python2.7/dist-packages/celery/bin/celery.py", line 769, in execute_from_commandline
    super(CeleryCommand, self).execute_from_commandline(argv)))
  File "/usr/local/lib/python2.7/dist-packages/celery/bin/base.py", line 307, in execute_from_commandline
    return self.handle_argv(self.prog_name, argv[1:])
  File "/usr/local/lib/python2.7/dist-packages/celery/bin/celery.py", line 761, in handle_argv
    return self.execute(command, argv)
  File "/usr/local/lib/python2.7/dist-packages/celery/bin/celery.py", line 693, in execute
    ).run_from_argv(self.prog_name, argv[1:], command=argv[0])
  File "/usr/local/lib/python2.7/dist-packages/celery/bin/worker.py", line 179, in run_from_argv
    return self(*args, **options)
  File "/usr/local/lib/python2.7/dist-packages/celery/bin/base.py", line 270, in __call__
    ret = self.run(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/celery/bin/worker.py", line 212, in run
    state_db=self.node_format(state_db, hostname), **kwargs
  File "/usr/local/lib/python2.7/dist-packages/celery/worker/__init__.py", line 100, in __init__
    self.setup_instance(**self.prepare_args(**kwargs))
  File "/usr/local/lib/python2.7/dist-packages/celery/worker/__init__.py", line 131, in setup_instance
    signals.worker_init.send(sender=self)
  File "/usr/local/lib/python2.7/dist-packages/celery/utils/dispatch/signal.py", line 166, in send
    response = receiver(signal=self, sender=sender, **named)
  File "/home/bkehoe/workspace/celery_test/test/tasks.py", line 16, in init
    print args, kwargs
  File "/usr/local/lib/python2.7/dist-packages/celery/worker/__init__.py", line 329, in __repr__
    self=self, state=self.blueprint.human_state(),
AttributeError: 'NoneType' object has no attribute 'human_state'

```

**Comment by bahdad** [comment]
Hello @ask 
I'm still having this error when starting my Celery app in the Docker container...
`2016-04-19 18:27:19,848 ERROR 'NoneType' object has no attribute 'human_state' 
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/raven/utils/serializer/manager.py", line 76, in transform
    return repr(value)
  File "/usr/local/lib/python2.7/dist-packages/celery/worker/__init__.py", line 329, in __repr__
    self=self, state=self.blueprint.human_state(),
AttributeError: 'NoneType' object has no attribute 'human_state'`
Django 1.6.5
Celery 3.1.23

**Comment by ask** [comment]
Thanks!  Seems this was already was already fixed in master, but not in 3.1 so I added a fix to the 3.1 branch.

**Comment by bahdad** [comment]
Great! Thanks for quick response and for the Celery as well! 👍 
Looking forward to new release with this fix 😄
